roles/hosted_engine_setup/examples/required_networks_fix.yml

```diff
@@ -0,0 +1,56 @@
+---
+# This is an example for a hook to fix restore-from-file errors
+# due to missing required networks.
+#
+# If you have an existing hosted-engine setup, and the Default cluster
+# has some required network, then if you take a backup and try to restore
+# it with '--restore-from-file', the deployment process cannot know which
+# host nic should be attached to the required network, and so activating
+# the host will fail. This will prompt the user to manually handle the
+# situation via the engine web admin UI.
+#
+# If you already know that beforehand, and want to automate restoration,
+# you can copy current file, edit as needed, and place it in:
+# /usr/share/ansible/collections/ansible_collections/ovirt/ovirt/roles/hosted_engine_setup/hooks/after_add_host/
+# File name should end with '.yml'.
+#
+# For more details, see (also):
+# https://docs.ansible.com/ansible/latest/collections/ovirt/ovirt/ovirt_host_network_module.html
+# https://docs.ansible.com/ansible/latest/collections/ovirt/ovirt/ovirt_host_module.html
```

Comment:
The latest documentation is for version 2.10 and we are using 2.9.
I didn't check if there is any difference between them.

Reply:
I generally prefer to link to latest.

That said, these modules are now maintained right here, in ovirt-ansible-collection, so it should continue working when we make changes.

I did wonder what ansible are going to do about documentation when splitting stuff out to collections, not sure.